src/client/api/quoteSelector.ts

```diff
@@ -82,4 +82,6 @@ const HOME_INSURANCE_TYPES = [
 export const isHomeContentsOrHouse = (quote: BundledQuote) =>
   HOME_INSURANCE_TYPES.includes(quote.data.type)
 
-export const getSubType = (quote: BundledQuote) => quote.data.subType
+type QuoteDataSubTypes = 'RENT' | 'OWN' | 'STUDENT_RENT' | 'STUDENT_OWN'
```

Comment:
Will this conflict with car that adds additional sub types? Right, @simonauner?

Reply:
I can also set it to string for the time being. Since it comes from the data blob it's not typed

Reply:
It is correct that we use sub-types also for Car, but this doesn't really seem to matter since when Car sends different types in runtime TS won't know about it anyway. 

But if we were to add some logic based on any Car value in the future we need to add it. But as Mehdi says, we're trying to type values in an `any` object anyway, so it's just developer ergonomics 🤷